README.md

```diff
@@ -0,0 +1,197 @@
+# GCP Dataflow Example Project
+
+## Introduction
+
+This is a simple stream processing example project, written in Scala and using Google Cloud Platform's services/APIs:
+
+- Dataflow for data processing
+- Pub/Sub for communication
+- Bigtable for storage
+
+**Running this requires a Google account and enabling GCP services, which will incur certain costs**
+
+We assume you have [pyinvoke](http://www.pyinvoke.org/) installed, as well as the [Google Cloud Python SDK](https://pypi.python.org/pypi/google-cloud), in order for the helper script to work.
```

Comment:
In the end, did you check for compatibility with python 2? It might be worth mentioning if it's not supported.